library/include/rocblas-auxiliary.h

```diff
@@ -19,7 +19,7 @@
 /* ============================================================================================ */
 /*! \brief  indicates whether the pointer is on the host or device. currently HIP API can only recoginize the input ptr on deive or not
 can not recoginize it is on host or not */
-ROCBLAS_EXPORT rocblas_mem_location
+ROCBLAS_EXPORT rocblas_pointer_mode
 rocblas_get_pointer_location(void *ptr);
```

Comment:
How about  rocblas_get_mode_of_pointer  ?

Then the functions would be
rocblas_get_pointer_mode, rocblas_set_pointer_mode, rocblas_get_mode_of_pointer